File: PHP/pertemuan7/angka.php
<?php
$a = readline("masukan angka pertama:");
$b = readline("masukan angka kedua:");

if ($a>$b)
    echo "$a lebih besar dari $b\n";
elseif ($a<$b)
    echo "$a lebih kecil dari $b\n";
else
    echo "keduanya sama besar\n";

echo"apakah keduanya sama?";
echo ($a==$b)? "true\n":"false\n";
?>

File: PYTHON/pertemuan7/angka.py
a = int(input("masukan angka pertama:"))
b = int(input("masukan angka kedua:"))

if a > b: 
  print(a,"lebih besar dari",b)
elif a<b:
  print(a,"lebih kecil dari",b)
else:
  print("keduanya sama besar")

print("apakah keduanya sama?",a==b)